mobile/src/main/java/com/gelakinetic/mtgfam/helpers/DeckListImporter.java

```diff
@@ -1,3 +1,22 @@
+/*
+ * Copyright 2017 Adam Feinstein
```

Comment:
If you want to add your name or handle to the files you modified, you are more than welcome to.